### ISSUE TITLE
Clarify Python 3.5 support in v4.0.0

### ISSUE THREAD

**Issue by PeterJCLaw** [chardet/chardet#215]
Hi,
Thanks for maintaining this project.
I'm (unfortunately) still using Python 3.5 in a few places and was looking at v4.0. The release seems to indicate that that version doesn't support Python 3.5, yet I can't actually see where in the changes it was dropped (the other versions being dropped are fairly clear) and the published packages will happily install on 3.5.
Could you clarify whether 3.5 is supported in v4? If it isn't, it might be worth putting something on PyPI to warn users who haven't looked at the releases (even just a link to the github releases page might be handy!)
Thanks!

**Comment by Nirzak** [comment]
> Hi,
> Thanks for maintaining this project.
> I'm (unfortunately) still using Python 3.5 in a few places and was looking at v4.0. The release seems to indicate that that version doesn't support Python 3.5, yet I can't actually see where in the changes it was dropped (the other versions being dropped are fairly clear) and the published packages will happily install on 3.5.
> Could you clarify whether 3.5 is supported in v4? If it isn't, it might be worth putting something on PyPI to warn users who haven't looked at the releases (even just a link to the github releases page might be handy!)
> Thanks!

Hi, Just take a look at the 48th line of setup.py here https://github.com/chardet/chardet/blob/master/setup.py it's clearly mentioned that `python_requires=">=3.6",` So, yes it's not supported on python 3.5

**Comment by PeterJCLaw** [comment]
Yeah, I'm aware that on `master` that change has been made, however in the tagged (and published) version at 4.0.0 that line still reads:
``` python
      python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*",
```
https://github.com/chardet/chardet/blob/4.0.0/setup.py#L51

This is the root of my query -- since `chardet` does indeed install on 3.5 (and 2.7) at version 4.0.0 it's not clear whether the support is really gone. Certainly it would be easy for users to accidentally upgrade without seeing the release notes (as I nearly did).

Maybe it would make sense to release a `4.0.0-post1` with that line fixed and something clearly in the README about 4.x dropping 3.5/2.7 support? That would increase visibility, though I think unless there's a way to unrelease the `4.0.0` package on PyPI there may not be a way to solve this completely?

**Comment by Nirzak** [comment]
Yup may be a re-release will fix this issue. i have also seen that on the pypy.org directory it's still showing "Requires: Python >=2.7"

**Comment by Nirzak** [comment]
Mentioning @dan-blanchard to take a look.

**Comment by domdfcoding** [comment]
The change to be Python 3.6+ was made in https://github.com/chardet/chardet/pull/205, which came after the 4.0.0 release. If nothing changes that will make it into the 5.0.0 release
